mysql-test/suite/rpl/t/rpl_mdev12179.test

```diff
@@ -70,9 +68,7 @@ SELECT * FROM t1 ORDER BY a;
 # Let the slave mysqld server start again.
 # As we are restarting, also take the opportunity to test --gtid-pos-auto-engines
 --echo *** Restart server with --gtid-pos-auto-engines=innodb,myisam ***
---append_file $MYSQLTEST_VARDIR/tmp/mysqld.2.expect
-restart: --skip-slave-start=0 --gtid-pos-auto-engines=innodb,myisam
-EOF
+--write_line "restart: --skip-slave-start=0 --gtid-pos-auto-engines=innodb,myisam" $MYSQLTEST_VARDIR/tmp/mysqld.2.expect
```

Comment:
write_line does not do the same thing as append_file. write_line overwrites, but append does not